docs/getting-started.md

```diff
@@ -1,372 +1,270 @@
+an execution jail, set up by the [`jailer`](../src/jailer/) binary. This is how
+our [integration test suite](#running-the-integration-test-suite) does it. This
+guide will not use the [`jailer`](../src/jailer/).
 
 ```bash
-git clone https://github.com/firecracker-microvm/firecracker
-```
+ARCH="$(uname -m)"
 
-All development happens on the main branch and we use git tags to mark
-releases. If you are interested in a specific release (e.g. v0.10.1), you can
-check it out with:
+# Download a linux kernel binary
+wget https://s3.amazonaws.com/spec.ccfc.min/img/quickstart_guide/${ARCH}/kernels/vmlinux.bin
 
-```bash
-git checkout tags/v0.10.1
-```
+# Download a rootfs
+wget https://s3.amazonaws.com/spec.ccfc.min/ci-artifacts/disks/${ARCH}/ubuntu-18.04.ext4
 
-### Building Firecracker
+# Download the ssh key for the rootfs
+wget https://s3.amazonaws.com/spec.ccfc.min/ci-artifacts/disks/${ARCH}/ubuntu-18.04.id_rsa
 
-Within the Firecracker repository root directory:
+# Set user read permission on the ssh key
+chmod 400 ./ubuntu-18.04.id_rsa
 
-1. With the __default__ musl target:
+# Clone the firecracker repository
+git clone https://github.com/firecracker-microvm/firecracker
 
-   ```tools/devtool build```
+# Start docker
+sudo systemctl start docker
 
-1. (__Experimental only__) using the gnu target:
+# Build firecracker
+#
+# It is possible to build for gnu, by passing the arguments '-l gnu'.
+#
+# This will produce the firecracker and jailer binaries under
+# `./firecracker/build/cargo_target/${toolchain}/debug`.
+#
+sudo ./firecracker/tools/devtool build
 
-   ```tools/devtool build -l gnu```
+API_SOCKET="./firecracker.socket"
 
-This will build and place the two Firecracker binaries at:
+# Remove API unix socket
+rm -f $API_SOCKET
 
-- `build/cargo_target/${toolchain}/debug/firecracker` and
-- `build/cargo_target/${toolchain}/debug/jailer`.
+# Run firecracker
+./firecracker/build/cargo_target/${ARCH}-unknown-linux-musl/debug/firecracker \
+    --api-sock "${API_SOCKET}"
+```
 
-If you would like to test a new feature and work with
-dependencies on libraries located in private git repos, you
-can use the `--ssh-keys` flag to specify the paths to your
-public and private SSH keys on the host. Both of them are
-required for git authentication when fetching the repositories.
+In a new terminal (do not close the 1st one):
 
 ```bash
-tools/devtool build --ssh-keys ~/.ssh/id_rsa.pub ~/.ssh/id_rsa
+TAP_DEV="tap0"
+TAP_IP="172.16.0.1"
+MASK_SHORT="/30"
+
+# Setup network interface
+sudo ip link del "$TAP_DEV" 2> /dev/null || true
+sudo ip tuntap add dev "$TAP_DEV" mode tap
+sudo ip addr add "${TAP_IP}${MASK_SHORT}" dev "$TAP_DEV"
+sudo ip link set dev "$TAP_DEV" up
+
+# Enable ip forwarding
+sudo sh -c "echo 1 > /proc/sys/net/ipv4/ip_forward"
+
+# Set up microVM internet access
+sudo iptables -t nat -D POSTROUTING -o eth0 -j MASQUERADE || true
+sudo iptables -D FORWARD -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT \
+    || true
```

Comment:
nit: remove the new line as well?

Reply:
The newline is needed to conform to [`mdl`](https://github.com/markdownlint/markdownlint) column max width.